<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>R12</class>
 <widget class="QMainWindow" name="R12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super particle viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cbRadiationType">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbSumR">
        <property name="text">
         <string>Sum all radii</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSingleR">
        <property name="text">
         <string>Individual radii</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblRIndex">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>[Index]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblR">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>[Radius]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="sliderRadius">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbSuper">
      <property name="title">
       <string>Super particle</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblParam1Name">
           <property name="text">
            <string>Parameter 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblParam1">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblParam2Name">
           <property name="text">
            <string>Parameter 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblParam2">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lbl">
           <property name="text">
            <string>Energy:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSuperEnergy">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Pitch angle:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSuperPitch">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnMark">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mark super-particle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlotRadialProfile">
        <property name="text">
         <string>Plot radial profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Save figure</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
